### DISCUSSION TITLE
Problems with homepage

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2977] [General Questions & Discussion] [answered]
Hi, just installed a new hubs server (hubs personal/aws) and found that the images in the settings are not displaying correctly. The Application logo is not centered and the fav icon is not showing. Im getting really frustrated with this because I really dont want to reinstall everything because of these two small issues. Any help would be appreciated.

**Comment by ocularvr** [comment]
Having to reinstall the whole stack because someone thought that hubs always works out of the box. Well, it doesn't. Something as stupid as an unaligned app image on the home page can waste 4 hours.

**Comment by robinkwilson** [comment, marked as answer]
Thanks for letting us know what you're seeing on your homepage especially making other threads that pinpoint specific examples.

Can you give me more information on what you're seeing? You may need to cache refresh the page to see your changes (on my mac it's **command + R**)

I tested a favicon.ico and it shows up for me. 

<img width="80" alt="image" src="https://user-images.githubusercontent.com/29560735/92637753-25b33200-f28e-11ea-93c9-e12c71c5735a.png">

Linking the other discussions on this:
https://github.com/mozilla/hubs/discussions/2991 - re: text translations
https://github.com/mozilla/hubs/discussions/2978 - re: logo alignment

**Reply by ocularvr** [reply]
Dear Robin,
First and foremost thank you for the time and feedback, I know that you are doing the best and I and every one here is really appreciative of all of your efforts. Regarding my issue, yeah there are loads of things that are happening with our setup, its the production one and we had the exact same issues on the development server (different aws/ip/etc). So whats happening with our homepage is that changes like the headline, description and so on do not get saved. We tried everything imaginable and are really stressed because this is the second time its happening. Have a look, https://ourworld.camera/ the text appears then disappears.

**Reply by robinkwilson** [reply]
Wow very bizarre, I haven't seen this behavior before. Can you email hubs@mozilla.com? Link this thread and mention me, I'd like to get to the bottom of this. 

Quick question: do you have a custom client deployed?

**Comment by ocularvr** [comment]
Thanks, your message gave me a buzz this morning :) (In the UK, buzz means
a big positive vibe! - I know out there it could mean something else!!!! :))

Regarding the issue, yeah I have two servers

https://ocularvr.net

https://ourowlrd.camera

The behavior is the same in both, the texts in the home page are not
visible due to layers moving around and other issues.

They are both AWS, they are both Hubs Personal, and they are both in
Virginia.

I am not using a custom client, I'm using an off the shelf stack.

Here is what's happening with both, the changes I make to things like
title, etc do not take effect. I have attached a screenshot.

Thank you

**Reply by robinkwilson** [reply]
Do you have any custom JS or CSS running? It looks like it loads the text, then something changes in the page after the DOM loads.
On ocularvr.net is that image the Logo image? (Whichever it is in the Admin Panel)

**Comment by ocularvr** [comment]
I have nothing running on that installation of hubs, it's out of the box.

**Reply by robinkwilson** [reply]
Do you have any fields filled in on any of the tabs in **Server Settings**? 
Then which fields in **App Settings**?

**Reply by ocularvr** [reply]
Nothing in server setup. I reinstalled the application on another server and it seems a bit more stable. 
Thanks for the feedback, I really appreciate it.
